queries/gdscript/highlights.scm

```diff
@@ -23,16 +29,13 @@
   "const" @type.qualifier
   (name) @constant)
 
-((identifier) @variable.builtin
-  (#eq? @variable.builtin "self"))
+(expression_statement (string) @comment @spell)
```

Comment:
Are strings here actually comments?

Reply:
Yes, strings as standalone statements are valid comments. Here's the PR to verify: https://github.com/godotengine/godot/pull/72971